index.html

```diff
@@ -1032,7 +1032,7 @@ <h4>serializeDerivedProofValue</h4>
 array of mandatory indexes (|mandatoryIndexes|), an array of
 selective indexes (|selectiveIndexes|), and a BBS presentation header
 (|presentationHeader|), the |featureOption| indicator, and possibly a
-|pseudonym| and |L| value depending on the |featureOption| value.
+|pseudonym| and/or |lengthBBSMessages| value depending on the |featureOption| value.
```

Comment:
Should probably include something about what `|featureOption| value` means there will be `a |pseudonym|`, `a |lengthBBSMessages|`, or both.
```suggestion
depending on the |featureOption| value, a |pseudonym| and/or |lengthBBSMessages| value.
```